controllers/GET-controllers.js

```diff
@@ -38,7 +40,10 @@ const getUsers = ((req, res, next) => {
 const getArticles = ((req, res, next) => {
 
 
-    fetchArticles().then((articles) => {
+    const { sortby, order, topic } = req.query;
+    const commentCount = req.query.hasOwnProperty('comment_count') ? 1 : 0;
+
```

Comment:
All requests to this endpoint should be including the comment count as standard. Looks like the SQL query is doing all this work for you already, so you can remove anything connected to the old comment count query.

Reply:
It has been removed from controllers as well